PerformanceCalculatorGUI/Screens/ProfileScreen.cs

```diff
@@ -207,7 +206,8 @@ private void calculateProfile()
                 var playcountBonusPP = (totalLivePP - nonBonusLivePP);
                 totalLocalPP += playcountBonusPP;
 
-                user.Value.Statistics.PP = totalLocalPP;
+                userPanel.livePp.Value = totalLivePP;
```

Comment:
Better wrap this in ```Schedule( () => { ... } );``` to avoid crashes due to UI being updated from other threads.